app/DataMapper/DefaultSettings.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\DataMapper;
+
+use App\Models\Client;
+
+class DefaultSettings
+{
+
+	public static $per_page = 20;
+
+	public static function userSettings() : \stdClass
+	{
+		return (object)[
+	        class_basename(Client::class) => self::clientSettings(),
+	    ];
+	}
+
+	private static function clientSettings() : \stdClass
+	{
+		
+		return (object)[
```

Comment:
What do you think removing __checkbox and __component:client-actions from the settings. We wouldn't want the user to be able to remove the columns so we know we can always show them.

Reply:
yep.. initially I thought it would be useful, but it probably may cause issues with some users.